src/shared/modules/connections/connectionsDuck.ts

```diff
@@ -630,7 +638,66 @@ export const connectionLostEpic = (action$: any, store: any) =>
     .filter(() => inWebEnv(store.getState()) && isConnected(store.getState()))
     .throttleTime(5000)
     .do(() => store.dispatch(updateConnectionState(PENDING_STATE)))
-    .mergeMap(() => {
+    .mergeMap((action: any) => {
+      if (action.error.code === 'Neo.ClientError.Security.TokenExpired') {
```

Comment:
we're planning to refactor so this code is not duplicated, right?

Reply:
yes